pyproject.toml

```diff
@@ -6,12 +6,32 @@ authors = ["CFA"]
 license = "Apache-2.0"
 readme = "README.md"
 packages = [{include = "forecasttools"}]
+include = [
+    { path = "forecasttools/location_table.csv", format = "sdist" },
+    { path = "forecasttools/location_table.csv", format = "wheel" },
+    { path = "forecasttools/example_flusight_submission.csv", format = "sdist" },
+    { path = "forecasttools/example_flusight_submission.csv", format = "wheel" },
+    { path = "forecasttools/example_flu_forecast.nc", format = "sdist" },
+    { path = "forecasttools/example_flu_forecast.nc", format = "wheel" },
+    { path = "forecasttools/nhsn_hosp_COVID.csv", format = "sdist" },
+    { path = "forecasttools/nhsn_hosp_COVID.csv", format = "wheel" },
+    { path = "forecasttools/nhsn_hosp_flu.csv", format = "sdist" },
+    { path = "forecasttools/nhsn_hosp_flu.csv", format = "wheel" },
+]
+
 
 [tool.poetry.dependencies]
 python = "^3.12"
 pre-commit = "^3.8.0"
 polars = "^1.8.2"
 arviz = "^0.19.0"
+ipykernel = "^6.29.5"
+requests = "^2.32.3"
+numpyro = "^0.15.3"
+diffrax = "^0.6.0"
+patsy = "^0.5.6"
```

Comment:
are these definitely deps?

Reply:
No. I have in the PR description, nestled among the tasks:

> `poetry` dependency grouping.

which needs to be done. The current package deps. are bloated and need to be grouped.

Reply:
Some of the other things in that tasks list could wait for a separate PR, but I think that one is essential for landing this PR. We should prevent having a `pyproject` unnecessary deps merged to main if possible.

Reply:
Agree. Thank you for the calibration of importance. I am updating the task list right now.